wake-lock/wakelock-object-is-independent.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WakeLock object is independent</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<script>
+
+function make_script() {
+  return '<script>' +
+         '  let iframeWakeLock, iframeRequest;' +
+         '  window.onmessage = message => {' +
+         '    if (message.data === "INIT") {' +
+         '      navigator.getWakeLock("screen")' +
+         '      .then(wakeLock => {' +
+         '        iframeWakeLock = wakeLock;' +
+         '        parent.postMessage(iframeWakeLock.active, "*");' +
+         '      });' +
+         '    } else if (message.data === "ACQUIRED") {' +
+         '      iframeRequest = iframeWakeLock.createRequest();' +
+         '      parent.postMessage(iframeWakeLock.active, "*");' +
+         '    } else if (message.data === "RELEASED") {' +
+         '      iframeRequest.cancel();' +
+         '      parent.postMessage(iframeWakeLock.active, "*");' +
+         '    }' +
+         '  };' +
+         '<\/script>';
+}
+
+function load_iframe(src) {
+  return new Promise(resolve => {
+    const iframe = document.createElement("iframe");
+    iframe.onload = () => { resolve(iframe); };
+    iframe.srcdoc = src;
+    iframe.style.display = "none";
+    document.documentElement.appendChild(iframe);
+  });
+}
+
+function wait_for_message(iframe) {
+  return new Promise(resolve => {
+    self.addEventListener("message", function listener(e) {
+      if (e.source === iframe.contentWindow) {
+        resolve(e.data);
+        self.removeEventListener("message", listener);
+      }
+    });
+  });
+}
+
+promise_test(async t => {
+  let wakeLock1 = await navigator.getWakeLock("screen");
+  let request1 = wakeLock1.createRequest();
+  let wakeLock2 = await navigator.getWakeLock("screen");
+  let iframe, request2;
+  return load_iframe(make_script())
+  .then(ifr => {
+    iframe = ifr;
+    iframe.contentWindow.postMessage('INIT', '*');
+    return wait_for_message(iframe);
+  })
+  .then(message => {
```

Comment:
Nit: the word `message` makes it sound like it's going to be a string. Maybe rename it to reflect that it's a boolean.